service/worker/esanalyzer/workflow.go

```diff
@@ -146,6 +148,47 @@ func (w *Workflow) getWorkflowVersionQuery(domainName string) (string, error) {
     `, domain.GetInfo().ID), nil
 }
 
+func (w *Workflow) getWorkflowTypePinotQuery(domainName string) (string, error) {
+	domain, err := w.analyzer.domainCache.GetDomain(domainName)
+	if err != nil {
+		return "", err
+	}
+	// exclude uninitialized workflow executions by checking whether record has start time field
+	// there's a "LIMIT 10" because in ES, Aggr clause by default returns the top 10 results
+	return fmt.Sprintf(`
+SELECT WorkflowType, COUNT(*) AS count
+FROM %s
+WHERE DomainID = '%s'
```

Comment:
nit: `%q` to replace `'%s'` according to https://pkg.go.dev/fmt

Reply:
For strings, %q returns a double-quoted string safely escaped with Go syntax, but in Pinot, Where DomainID = "<domain-id>" doesn't work. It has to be single quoted.